### PR TITLE
Add basic CircleCI config

### PR DESCRIPTION
**Pull request by indyfree**
Towards: https://github.com/giantswarm/giantswarm/issues/4848

Adds Circle to prometheus to enable git releases in the future.

### CODE REVIEW

**Review by puja108 — APPROVED** [review]
LGTMIIWFY

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
+# How to contribute
+
+PROJECT is Apache 2.0 licensed and accepts contributions via GitHub pull
```
Comment (rossf7) [inline CONTRIBUTING.md]: Not sure if we need this file or not? architect doesn't check for it. If we do add we should fix the project name.

**Review by rossf7 — APPROVED** [review]
LGTM just a nit on the paperwork.

File: CONTRIBUTING.md (hunk 1 above)
Comment (indyfree) [reply CONTRIBUTING.md]: I think it makes sense since this is one of our repos, that actually got/get contributions, WDYT?  I'll change the project name

Comment (rossf7) [reply CONTRIBUTING.md]: 👍 Good call to add it. Makes sense to have on repos where we get contributions.

**Pull request merged**